src/plugins/data/public/ui/dataset_selector/dataset_table.tsx

```diff
@@ -86,7 +86,27 @@ export const DatasetTable: React.FC<DatasetTableProps> = (props) => {
       <EuiBasicTable
         items={dataStructures}
         itemId="id"
-        columns={[{ field: 'title', name: 'Name' }]}
+        columns={[
+          {
+            field: 'title',
+            name: 'Name',
+            textOnly: true,
+            render: (title: string, item: DataStructure) => {
+              return (
+                <>
+                  <EuiText size="s" className="datasetTable__itemTitle">
+                    {title}
+                  </EuiText>
+                  {item.description && (
+                    <EuiText size="s" color="subdued" className="datasetTable__itemDescription">
```

Comment:
should the title and description have the same size? can imagine if each structure has a description it can get lengthy

Reply:
they are the same size?
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/0404e5836c5740a73cf71a8decc820ac9870867c/src/plugins/data/public/ui/dataset_selector/dataset_table.tsx#L97-L98

Reply:
title and description are both small size text `<EuiText size="s"`, maybe use `xs` for description?

Reply:
makes sense, added truncation
![image](https://github.com/user-attachments/assets/b89855cc-82bf-42c0-93ee-7ff04700d670)